pycode/memilio-epidata/memilio/epidata_test/test_epidata_getCaseData.py

```diff
@@ -176,9 +180,12 @@ def test_get_case_data_read(self):
         file_with_path = os.path.join(directory, file)
 
         with self.assertRaises(FileNotFoundError) as error:
-            gcd.get_case_data(read_data, file_format, out_folder, no_raw,
-                              impute_dates, make_plot, moving_average,
-                              split_berlin, rep_date)
+            gcd.get_case_data(
+                read_data=read_data, file_format=file_format,
+                out_folder=out_folder, no_raw=no_raw, start_date=start_date,
+                end_date=end_date, impute_dates=impute_dates,
+                make_plot=make_plot, moving_average=moving_average,
+                split_berlin=split_berlin, rep_date=rep_date)
```

Comment:
Of course you can change the order if you use the names but this is now messing things up using an arbitrary order of named arguments...

Maybe it is then still the better option to bring the arguments of lower importance to the end? In particular split_berlin and rep_date? see comment in get_case_data()

Reply:
The order is now:
read_data, file_format, out_folder, no_raw, start_date, end_date, impute_dates, moving_average, make_plot, (split_berlin, rep_date (only for case data))